include/wobbly/geometry.h

```diff
@@ -0,0 +1,352 @@
+/*
+ * src/geometry.h
+ *
+ * Provides geometric types and trait overloads
+ * for 2D points and boxes.
+ *
+ * See LICENCE.md for Copyright information
+ */
+#ifndef WOBBLY_GEOMETRY_H
+#define WOBBLY_GEOMETRY_H
+
+#include <wobbly/geometry_traits.h>
+
+namespace wobbly
+{
+    namespace geometry
+    {
+        namespace detail
+        {
+            struct CopyablePV
+            {
+                public:
+
+                    CopyablePV (CopyablePV const &copy) = default;
+                    CopyablePV & operator= (CopyablePV const &copy) = default;
+            };
+
+            struct NoncopyablePV
+            {
+                public:
+
```

Comment:
This and the above `public` have a newline below but not the `PointViewCopyabilityBase` class. I prefer it without the newline, but I am not sure you just copied it from somewhere and wanted to keep it like this for some reason.

Reply:
Ah, I should adjust `PointViewCopyabilityBase`. I am pretty sure there is meant to be a newline.